contrib/kind-common.sh

```
@@ -1167,13 +1245,20 @@ clone_frr() {
   [ -d "$FRR_TMP_DIR" ] || {
     mkdir -p "$FRR_TMP_DIR" && trap 'rm -rf $FRR_TMP_DIR' EXIT
     pushd "$FRR_TMP_DIR" || exit 1
-    git clone --depth 1 --branch $FRR_K8S_VERSION https://github.com/metallb/frr-k8s
+    git clone --no-tags --single-branch --branch main https://github.com/metallb/frr-k8s
+    pushd frr-k8s
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)